testdata/simple_wk_dataset/datasource-properties.json

```diff
@@ -4,7 +4,7 @@
         "team": ""
     },
     "scale": [
-        1
+        1, 1, 1
```

Comment:
According to https://github.com/scalableminds/webknossos/blob/master/frontend/javascripts/types/schemas/datasource.schema.js, the scale always consists of 3 numbers.

Reply:
I don't know why this file contained such a scale. We already used to write `Tuple[float, float, float]` before this PR.
This should not have worked before because the scale would have been simply `(1,)`. I assume that the test cases which used this dataset never needed the scale.
The current implementation does not support this. Do you think this is necessary? I am not sure how to define a structuring hook for an individual attribute (instead of a class).